.github/workflows/build.yaml

```diff
@@ -34,9 +34,7 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.17.x'
-      - uses: imjasonh/setup-ko@v0.4
-        with:
-          version: tip
+      - uses: imjasonh/setup-ko@main
```

Comment:
This should still only install the latest release by default. If it doesn't that's a bug.

Reply:
what should I need to do exactly?

Reply:
```suggestion
      - uses: imjasonh/setup-ko@main
        with:
          version: tip
```

Reply:
ah, yes 🤦